lambda/main.go

```diff
@@ -61,7 +63,14 @@ func HandleRequest(ctx context.Context) error {
 		log.Printf("error while uploading exploit db file to s3. Bucket:%s, Region:%s,Error:%s", AWS_S3_BUCKET, AWS_S3_REGION, err)
 	}
 
-	return err
+	// Invalidate CloudFront cache for the uploaded file
+	err = invalidateCloudFrontCache(session)
+	if err != nil {
+		log.Printf("error while invalidating CloudFront cache. DistributionID:%s, Error:%s", CLOUDFRONT_DIST_ID, err)
+		return err
```

Comment:
Not required. You can change return nil in the end to return err

Reply:
updated @namandf